### PR TITLE
Add adbTimeout in the config.

### PR DESCRIPTION
**Pull request by intelradoux**
With too much devices there is timeout issue.

### CODE REVIEW

File: src/main/groovy/com/stanfy/spoon/gradle/SpoonRunTask.groovy [hunk 1]
```diff
@@ -114,7 +117,11 @@ class SpoonRunTask extends DefaultTask implements VerificationTask {
     if (testSize != TEST_SIZE_ALL) {
       // Will throw exception with informative message if provided size is illegal
       runBuilder.setTestSize(IRemoteAndroidTestRunner.TestSize.getTestSize(testSize))
-    }    
+    }  
+    
+    if (adbTimeout.toString().isInteger()){
```
Comment (roman-mazur) [inline src/main/groovy/com/stanfy/spoon/gradle/SpoonRunTask.groovy]: I do not fully understand what this statement should actually do... :) What is your intent? Do you want to make `adbTimeout` optional?

Comment (roman-mazur) [inline src/main/groovy/com/stanfy/spoon/gradle/SpoonRunTask.groovy]: Also nit: space required before `{`.

**Comment by intelradoux** [comment]
I don't know how to force parameter to be int so the statement has 2 objective : 
- adbTimeout can be optional
- I check adbTimeout to be an int before apply it.

**Comment by roman-mazur** [comment]
Merged with few corrections 3cadd80979abb0cb31447688472c546be4f938e1. Let me know whether a new snapshot works for you.

**Pull request merged**